File: FastClean/FastCleanAPI/Models/Marcacao.cs
using System.ComponentModel.DataAnnotations;

namespace FastCleanAPI.Models
{
    public class Marcacao
    {
        public int MarcacaoId { get; set; }
        public TipoImovel TipoImovel { get; set; }
        public TipoLimpeza TipoLimpeza { get; set; }
        public TipoAgendamento TipoAgendamento { get; set; }
        public int NumQuartos { get; set; }
        public int NumCasasDeBanho { get; set; }
        public bool Cozinha { get; set; }
        public bool Sala { get; set; }
        public string? HoraInicial { get; set; }
        public string? HoraFinal { get; set; }
        public string? Detalhes { get; set; }
        public string? DiaHora { get; set; }
        public virtual Cliente? Cliente { get; set; }
        public virtual Funcionario? Funcionario { get; set; }
        public bool MarcacaoAceitePeloFunc { get; set; }
        public bool MarcacaoAceitePeloCliente { get; set; }
        public bool Terminada { get; set; }
        public string? MoradaMarcacao { get; set; }
        public string? LatitudeMarcacao { get; set; }
        public string? LongitudeMarcacao { get; set; }
        public string? DuracaoTotal { get; set; }
        public int NumHorasPrevistas { get; set; }
        public double Total { get; set; }
        public bool CliReview{ get; set; }
        public bool FuncReview { get; set; }

    }
}

File: FastCleanApp/app/src/main/java/com/example/fastclean/Models/Marcacao/Marcacao.kt
package com.example.fastclean.Models.Marcacao


import java.io.Serializable

data class Marcacao(
    val marcacaoId : Int,
    val tipoImovel : String,
    val tipoLimpeza : String,
    val tipoAgendamento : String,
    val numQuartos : Int,
    val numCasasDeBanho : Int,
    val cozinha : Boolean,
    val sala : Boolean,
    val detalhes : String,
    val diaHora : String,
    val horaInicio: String,
    val horaFinal : String,
    val clienteID : Int,
    val cliente : String,
    val funcionarioID : Int,
    val funcionario: String,
    val marcacaoAceitePeloFunc : Boolean,
    val marcacaoAceitePeloCliente : Boolean,
    val terminada : Boolean,
    val numHorasPrevistas : Int,
    val moradaMarcacao : String,
    val latitudeMarcacao :String,
    val longitudeMarcacao : String,
    var reviewCliente : Boolean,
    var reviewFuncionario : Boolean,
    var total: Float,
    var estadoFuncionario: String
): Serializable
